index.html

```diff
@@ -12771,30 +12812,40 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
             </caption>
             <thead>
               <tr>
-                <th scope="col">Value</th>
+                <th scope="col">Value (keyword)</th>
+                <th scope="col">State</th>
                 <th scope="col">Description</th>
               </tr>
             </thead>
             <tbody>
               <tr>
                 <th class="value-name" scope="row">false</th>
+                <th class="value-name" scope="row">False</th>
                 <td class="value-description">The element supports being checked but is not currently checked.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">mixed</th>
+                <th class="value-name" scope="row">Mixed</th>
                 <!-- make sure this value remains synced with its counterpart in #aria-pressed -->
                 <td class="value-description">Indicates a mixed mode value for a tri-state checkbox or menuitemcheckbox.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">true</th>
+                <th class="value-name" scope="row">True</th>
                 <td class="value-description">The element is checked.</td>
               </tr>
               <tr>
-                <th class="value-name" scope="row"><strong class="default">undefined</strong> (default)</th>
+                <th class="value-name" scope="row">undefined, empty string ("")</th>
```

Comment:
We could reasonably drop the `undefined` keyword here:

```suggestion
                <th class="value-name" scope="row">empty string ("")</th>
```

If someone set `"undefined"` then it would map to the invalid value default, which is the undefined state. The only reason _not_ to do this is if you want the canonicalization steps to explicitly return `undefined` rather than the empty string.

The same is true for other properties, e.g. `aria-current`, and so on.